utils/github.py

```diff
@@ -25,3 +26,32 @@ def get_file_from_github(github_api_token, dependency_repo, filepath):
         logger.error("No file content")
 
     return base64.b64decode(file_content).decode()
+
+
+def construct_filepath(filepath):
+    # return the same thing if the file is in the local repo
+    if is_local_file(filepath):
+        return filepath
+
+    # extract "contracts..." substring (preceded by "/") from filepath
+    # e.g. "@openzeppelin/contracts-v4.4/utils/structs/EnumerableSet.sol" => "contracts-v4.4/utils/structs/EnumerableSet.sol"
+    regex = re.compile("(?<=\/)contracts.*")
```

Comment:
Can we avoid regex and just define stupid remappings in the config?

Reply:
I was thinking about it in this way:
1) our remapping is always a prefix, not an arbitrary substring in the middle
2) we can sort remappings based on prefix length and apply them starting from the longest ones, returning early if found
3) maybe it would be great to require every remapping ends with closing '/'

Doing all of this (or at least 1+2) would allow us to avoid regular expressions at all